src/Factory/AwsFactory.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace Aws\Factory;
```

Comment:
We should probably put this module's classes in a different namespace, since there is a [potential risk of collision](https://github.com/aws/aws-sdk-php/tree/master/src) in the future.